packages/flutter_tools/templates/plugin/pubspec.yaml.tmpl

```diff
@@ -4,11 +4,43 @@ version: 0.0.1
 author:
 homepage:
 
+# The following section is specific to Flutter.
 flutter:
   plugin:
     androidPackage: {{androidIdentifier}}
     pluginClass: {{pluginClass}}
 
+  # To add assets to your plugin, add an assets section, like this:
```

Comment:
Maybe s/plugin/plugin package/ to make the links below about packages make more sense